src/commands/bootstrap-aws-context-creds.yml

```diff
@@ -1,6 +1,6 @@
 description: |
   Each CI context can inject an AWS credential to the environment.
-  Unpack them all and combine into a single credentials file inside ~/.aws/credentials
+  Unpack and combine into a single creds file inside ~/.aws/credentials
```

Comment:
How about just remove 'creds'?  Abbreviations are harder for some folks to read and it's redundant with the actual filename.  Plus already mentioned above.